### PR TITLE
[WIP] 474 feature py3

### PR DESCRIPTION
**Pull request by duk3luk3**
PR opened mainly to coax travis

Initial PR:

- [ ] PR branch should be named `issuenum`-`fix`/`feature`/`cleanup`-`description`
- [ ] Code is split into logical commits
- [ ] Code has tests
- [ ] Final commit includes "Fixes #issue" in commit message

When all builds pass and a maintainer is happy with your PR, the "ready" label will be applied. Please complete these tasks then:

- [ ] Rebase onto develop
- [ ] Add changelog entry
- [ ] Remove this entire template section

### CODE REVIEW

**Comment by duk3luk3** [comment]
@muellni @Brutus5000 @Sheeo travis build works now (will migrate the https://github.com/duk3luk3/faf-pyclient-builddeps/ repo when I merge this), but I need help with the windows build - I have no idea why it doesn't work, I just copied @muellni's improved python2 appveyor build. Maybe it needs a different build environment.

**Comment by muellni** [comment]
I'll look into the appveyor build.

**Comment by muellni** [comment]
Appveyor builds an installer now, but it's not running yet. Let me know if you need more help.

**Comment by duk3luk3** [comment]
I get this in a pretty much pristine windows 7 virtualbox (no fa installed):

```
---------------------------
FAForever.exe - Entry Point Not Found
---------------------------
The procedure entry point ucrtbase.terminate could not be located in the dynamic link library api-ms-win-crt-runtime-l1-1-0.dll. 
---------------------------
OK   
---------------------------
```

**Comment by duk3luk3** [comment]
On my windows 7 desktop, it just doesn't start up. This is all that's written to the log:

```
2016-11-27 19:18:58,818 INFO     root                           FAF version: 0.11.60-128-g0015698  + 1063 Environment: production
```

File: src/client/_clientwindow.py [hunk 1]
```diff
@@ -647,66 +637,69 @@ def cleanup(self):
...
         self.state = ClientState.SHUTDOWN
 
-        self.progress.setWindowTitle("FAF is shutting down")
-        self.progress.setMinimum(0)
-        self.progress.setMaximum(0)
-        self.progress.setValue(0)
-        self.progress.setCancelButton(None)
-        self.progress.show()
+        progress = QtWidgets.QProgressDialog()
+        progress.setMinimum(0)
```
Comment (GrotheFAF) [inline src/client/_clientwindow.py:633]: why is setMinimun and setMaximum there twice?

Comment (Wesmania) [reply src/client/_clientwindow.py:633]: Yeah, once should be enough. We can make a commit to fix that with  'git commit --fixup <original commit sha>', and we'll meld it in once we start rebasing the branch.

**Pull request merged**